include/kllvm/binary/serializer.h

```diff
@@ -28,10 +28,10 @@ class serializer {
 public:
   enum flags {
     NONE = 0,
-    DROP_HEADER = 1,
-    DROP_ARITY = 2,
-    DROP_BOTH = 3,
-    NO_INTERN = 4,
+    DropHeader = 1,
+    DropArity = 2,
+    DropBoth = 3,
+    NoIntern = 4,
```

Comment:
Shouldn't this be `lower_case` based on  `readability-identifier-naming.EnumCase`?